.changeset/itchy-bears-jump.md

```diff
@@ -0,0 +1,24 @@
+---
+"@spruceid/ssx": minor
+"@spruceid/ssx-server": minor
```

Comment:
Love to see changesets used here! The notes you added are excellent. Are you sure these are not patch changes (from a versioning perspective)? Marking them as minor will bump the version on ssx and ssx-server to `0.2.0`.

Reply:
Yes, I think it's a minor because it's a feature that's being added.